v03_pipeline/lib/annotations/shared.py

```diff
@@ -8,33 +8,6 @@
 
 from v03_pipeline.lib.model import ReferenceGenome
 
-AMINO_ACIDS = [
-    'A',
-    'C',
-    'D',
-    'E',
-    'F',
-    'G',
-    'H',
-    'I',
-    'K',
-    'L',
-    'M',
-    'N',
-    'P',
-    'Q',
-    'R',
-    'S',
-    'T',
-    'V',
-    'W',
-    'Y',
-    'X',
-    '*',
-    'U',
-]
-AMINO_ACIDS_LOOKUP = hl.dict(hl.enumerate(AMINO_ACIDS, index_first=False))
```

Comment:
There's weird stuff buried in the amino acid string.

For example:

```
G/GPAPPPSLHSLPSX
```

or 

```
L/LGX
```

I think this is supposed to mean `L` -> `L` OR `G` or `X`?

Reply:
I just ripped out the enumification so we could get a run in and see what's there.

Reply:
can we run off this branch or do we need to merge this to master first?

Reply:
Yeah, I (already did) can run off this branch.  It's a PR into a branch anyways... no rush here.